metrics/core.go

```diff
@@ -1,119 +1,198 @@
 package metrics
 
 import (
-	"fmt"
-	"time"
-
 	"github.com/prebid/prebid-cache/config"
-	"github.com/rcrowley/go-metrics"
-	"github.com/sirupsen/logrus"
-	"github.com/vrischmann/go-metrics-influxdb"
+	influx "github.com/prebid/prebid-cache/metrics/influx"
+	prometheus "github.com/prebid/prebid-cache/metrics/prometheus"
+	"time"
 )
 
-type MetricsEntry struct {
-	Duration   metrics.Timer
-	Errors     metrics.Meter
-	BadRequest metrics.Meter
-	Request    metrics.Meter
+// Object to access metric engines
```

Comment:
Please update comment to Go conventions, something like:
```
Metrics provides access to metric engines.
```

Reply:
done